tests/browser/conflict-resolution.spec.ts

```
@@ -0,0 +1,171 @@
+import { test, expect } from '@playwright/test';
+
+test.describe('Conflict Resolution UI Walkthrough', () => {
+  test.beforeEach(async ({ page }) => {
+    await page.goto('http://localhost:3000');
+    await page.waitForSelector('.app-shell');
+
+    // Seed IndexedDB with a conflict gist and its metadata
+    await page.evaluate(async () => {
+      const dbRequest = indexedDB.open('d-o-gist-hub-db', 3);
+      await new Promise((resolve) => {
+        dbRequest.onsuccess = (e: any) => {
```

Biome (2.4.15)

[error] Unexpected any. Specify a different type.

(lint/suspicious/noExplicitAny)